### ISSUE TITLE
CT: 2 Lint Errors

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/molarity#64]
Molarity is failing lint in CT.  https://github.com/phetsims/molarity/commit/083cc3707ecf4f68c04b68338ae10871413ebd9a appears to be the culprit.

```
perennial : lint-everything
Failure to lint everything:
[4mRunning "lint-everything" task[24m

/data/share/phet/continuous-testing/snapshot-1553525043260/molarity/js/molarity/view/describers/SolutionDescriber.js
   59:9   error  'solidsString' is assigned a value but never used  no-unused-vars
  326:31  error  Unexpected trailing comma                          comma-dangle

✖ 2 problems (2 errors, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
Fatal error: 2 Lint Errors


Approximately 3/25/2019, 8:44:03 AM
```

**Comment by twant** [comment]
Thanks, Chris! Apologies for this -- will run `grunt lint` faithfully in the future!
